src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
+    private static final int startPositionForSubstringDate = 0;
```

Comment:
read about naming convention for constants and refactor them

Reply:
Agree, my bad(
I've read it before, just forgot. Tomorrow morning I will send fixed version.